.gitignore

```diff
@@ -1,2 +1,2 @@
-node_modules/
+node_modules
 test
```

Comment:
- [ ] The `test.md` file was not ignored in your project because its not named properly in the `.gitignore` file. Kindly name it exactly as the test file.
- To see how this works, you can comment on the `node_modules` folder, and you'll see thousands of files it contains that help with abstraction but not ideally needed to be used directly by us. We then ignore it to keep our code clean and memory neat 👍